woocommerce/woocommerce
# Log tracks events after they have been validated

### All Submissions:






### Changes proposed in this Pull Request:

Updates the logged events in the debugger log to only log after they have passed all tracks validation.

Previously this caught any events through the event properties filter, but this did not guarantee that the event was sanitized, validated, or would be sent to tracks as a pixel.

This method captures the pixel right before its sent.

Closes # .

### How to test the changes in this Pull Request:

1. Test some events that are recorded by remote request (e.g., task list related events such as completion of tasks, lists, or hiding the list)
2. Test some events that are recorded by footer pixel (e.g., WC settings view)
3. Under Status->Logs, check the latest tracks log and make sure your event name and all properties exist.

### Other information:




### FOR PR REVIEWER ONLY:
